### PR TITLE
AIFBR-6908: Update metadata for 20.10.4 models

### PR DESCRIPTION
**Pull request by irinel-birsan-uipath**
Update metadata for 20.10.3 models

### CODE REVIEW

**Comment by mohitseth-nitrr** [comment]
@irinel-birsan-uipath  please raise it against alpha

File: metadata/JapanInvoices__1__metadata.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+﻿{
+    "name":  "JapanInvoices",
```
Comment (vivekjhauipath) [inline metadata/JapanInvoices__1__metadata.json:2]: @irinel-birsan-uipath pls double check with PMs on name?
Would this be japanese invoices or JapanInvoices?

**Review by vivekjhauipath — DISMISSED** [review]
LGTM.
Just double check name for JapanInvoices

File: metadata/JapanInvoices__1__metadata.json (hunk 1 above)
Comment (irinel-birsan-uipath) [reply metadata/JapanInvoices__1__metadata.json:2]: You are right. I will double-check anyway.

Comment (irinel-birsan-uipath) [reply metadata/JapanInvoices__1__metadata.json]: Done!

**Review by vivekjhauipath — APPROVED** [review]
LGTM

**Comment by mohitseth-nitrr** [comment]
@irinel-birsan-uipath  lets not merge it till respective images are pushed to replicated  aif-core

**Comment by irinel-birsan-uipath** [comment]
> @irinel-birsan-uipath lets not merge it till respective images are pushed to replicated aif-core

Not sure I understand. Do you mean when the models are pushed to prod environment?

**Comment by vivekjhauipath** [comment]
> > @irinel-birsan-uipath lets not merge it till respective images are pushed to replicated aif-core
> 
> @mohitseth-nitrr Not sure I understand. Do you mean when the models are pushed to prod environment?

@mohitseth-nitrr : @irinel-birsan-uipath pushed all images to alpha yesterday. We should merge this now

**Comment by irinel-birsan-uipath** [comment]
> > > @irinel-birsan-uipath lets not merge it till respective images are pushed to replicated aif-core
> > 
> > 
> > @mohitseth-nitrr Not sure I understand. Do you mean when the models are pushed to prod environment?
> 
> @mohitseth-nitrr : @irinel-birsan-uipath pushed all images to alpha yesterday. We should merge this now

Yes, let's merge it.

**Comment by irinel-birsan-uipath** [comment]
@mohitseth-nitrr @adeepsri Please take a look to can merge it in alpha.

**Review by mohitseth-nitrr — APPROVED** [review]
LGTM

**Review by vivekjhauipath — APPROVED** [review]
LGTM

**Pull request merged**